src/main/java/org/cactoos/text/TextOf.java

```diff
@@ -317,23 +317,9 @@ public TextOf(final String input) {
      * Ctor.
      *
      * @param iterable The iterable to convert to string
-     * @todo #1461:30min We want constructors with {@code Iterable<?>} and
-     *  {@code Iterator<Character>} to have same behaviour (simply concatenate list of strings).
-     *  To do that, we should change {@code Iterable<?>} to {@code Iterable<Character>}
-     *  and delegate the {@link Iterator} one to the {@link Iterable} one. After that, we need to
-     *  modify other parts of cactoos that relied on this in order to preserve their behaviour
-     *  by using {@link Joined} and {@link Concatenated}.
-     */
-    public TextOf(final Iterable<?> iterable) {
-        super(
-            new Joined(
-                ", ",
-                new Mapped<>(
-                    Object::toString,
-                    iterable
-                )
-            )
-        );
+     */
+    public TextOf(final Iterable<Character> iterable) {
+        this(iterable.iterator());
```

Comment:
@DmitryBarskov actualy I think it was the opposite: the Iterator one should delegate to the iterable one :) if not the Text object would be callable only once and then iterator would end up consumed. So this means the Iterable one should have the behaviour that is currently in the Iterator one.